virt/vm_networking/virt-connecting-vm-to-sriov.adoc

```diff
@@ -0,0 +1,26 @@
+:_content-type: ASSEMBLY
+[id="virt-connecting-vm-to-sriov"]
+= Connecting a virtual machine to an SR-IOV network
+include::_attributes/common-attributes.adoc[]
+:context: virt-connecting-vm-to-sriov
+
+toc::[]
+
+You can connect a virtual machine (VM) to a Single Root I/O Virtualization (SR-IOV) network by performing the following steps:
+
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#nw-sriov-configuring-device_virt-connecting-vm-to-sriov[Configure an SR-IOV network device.]
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#nw-sriov-network-attachment_virt-connecting-vm-to-sriov[Configure an SR-IOV network.]
+. xref:../../virt/vm_networking/virt-connecting-vm-to-sriov.adoc#virt-attaching-vm-to-sriov-network_virt-connecting-vm-to-sriov[Connect the VM to the SR-IOV network.]
+
+[id="connecting-vm-to-sriov-network_virt-connecting-vm-to-sriov"]
+== Connecting a virtual machine to an SR-IOV network
+
```

Comment:
This H2 repeats the assembly title and as it currently stands is orphaned. Is there a reason to keep it around?